scripts/build-upload

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 
-[ -n "$GCLOUD_AUTH" ] || {
+# Check for AWS Credentials
+[ -n "AWS_SECRET_ACCESS_KEY" ] || {
```

Comment:
ah you forgot the `$ ` in front of `AWS_SECRET_ACCESS_KEY` AND `AWS_ACCESS_KEY_ID`. 
Same in log-upload.

Reply:
Done @ivan-claire